### DISCUSSION TITLE
Don't log health requests

### DISCUSSION THREAD

**Discussion by MatthiasWiesner** [imgproxy/imgproxy#1239] [Ideas]
Is it possible to switch off logging for the healthchecks?

Our imgproxy instances are monitored by an icinga instance. Requests are sent to the health endpoint every second. Unfortunately, these requests are also logged. This makes log monitoring very confusing.

**Comment by DarthSim** [comment]
Hey @MatthiasWiesner!
That's a good idea, and actually, I was thinking about this myself. The `latest` build doesn't log health-check requests/responses.
